src/ganted2radius.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/net/context"
+	"layeh.com/radius"
+	"layeh.com/radius/rfc2865"
+	"layeh.com/radius/rfc2866"
+)
+
+func archiveLog(src, dst string) error {
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer srcFile.Close()
+
+	dstFile, err := os.Create(dst)
```

Comment:
If we're logging for auditing reasons, wouldn't this overwrite existing log?

Reply:
Write some logic to rotate log files? And archive daily? like https://github.com/ustclug/light-server/blob/master/.docker/rootfs/etc/periodic/hourly/squid2radius#L6-L10

Reply:
Maybe we could implement with this logic:

0. Compress all existing access-\<datetime>.log to access-\<datetime>.log.zst
1. Rename access.log to access-\<datetime>.log
2. Ask access log logger to reopen access.log, which would let it just stop writing to the renamed access log
3. Process access-\<datetime>.log

Reply:
Implemented at new commit.